src/main/java/com/limechain/storage/DBRepository.java

```diff
@@ -191,13 +187,6 @@ public void commitTransaction() {
         //TODO: implement
     }
 
-    private byte[] getPrefixedKey(String key) {
-        return chainPrefix.concat(key).getBytes();
-    }
-    public byte[] removePrefixFromKey(byte[] key) {
-        return Arrays.copyOfRange(key, chainPrefix.length(), key.length);
-    }
-
```

Comment:
Was removed, because it's absolutely obsolete to add chain name as prefix, because we are using different db directories for the different chains